plugin.yml

```
@@ -4,10 +4,15 @@ author: https://github.com/joscha
 requirements:
   - bash
   - docker
+  - gpg
```

possibly:
```suggestion
  - gpg
  - git
```
with my suggestion